bears/vcs/git/GitCommitBear.py

```diff
@@ -359,7 +369,8 @@ def check_issue_reference(self, body,
                 reference = compiled_issue_ref_regex.fullmatch(issue)
                 if not reference:
                     yield Result(self, 'Invalid {} reference: '
-                                       '{}'.format(result_info, issue))
+                                       '{}'.format(
+                                        self.issueType, issue))
```

Comment:
either put `.format` on this line, or intend this line another 3 spaces.  pycodestyle isnt perfect - it will accept a lot of non-pep8 compliant indentations because it is quite a difficult problem to know the correct indentation level needed.  But the fact you have a one space indentation here should be a clue that it is wrong even if pycodestyle accepts it.